app/realtime.php

```diff
@@ -153,6 +190,8 @@ function () use ($register, $db, $cache) {
                 'value' => json_encode($payload)
             ]);
         } catch (\Throwable $th) {
+            call_user_func($logError, $th, "updateWorkerDocument");
+
```

Comment:
Can we move all the Console::error logs to the error callback?

Reply:
We can. Moved it.